command/operator_diagnose.go

```diff
@@ -588,17 +589,7 @@ SEALFAIL:
 	licenseCtx, licenseSpan := diagnose.StartSpan(ctx, "Check For Autoloaded License")
 	// If we are not in enterprise, return from the check
 	if !constants.IsEnterprise {
```

Comment:
I like the idea of leaving `constants.IsEnterprise` alone for now, but we might think about removing it later...

Reply:
Mostly its used in PKI and in Core, which isn't all that surprising. I think perhaps I'd suggest we remove it now.